pyproject.toml

```diff
@@ -211,6 +217,9 @@ doctest_subpackage_requires = [
     "docs/io/fits/usage/image.rst = numpy<1.25",  # NUMPY_LT_1_25 (Issue 14545)
     "docs/io/fits/usage/unfamiliar.rst = numpy<1.25",  # NUMPY_LT_1_25 (Issue 14545)
 ]
+markers = [
+    "mpl_image_compare",
```

Comment:
Marker which is not specifically defined, that was caught by `--strict-markers`.

Reply:
I am not familiar with this one, maybe @astrofrog or @Cadair can confirm.

Reply:
All this is doing is registering a mark that is already in use in our codebase. Ideally, it either needs to be registered here or removed from the code if it is not needed anymore.

Reply:
I think this is so that there is no warning if someone runs the visualization tests and doesn't have pytest-mpl installed

Reply:
So in that case, there is no reason for us to not have the marker registered. The `--strict-markers` is mostly to guard against users trying to run with non-existent makers (like making a typo in the CLI); however, it also guards against typos in specifying the marker string on a given test.

Reply:
If we register the marker then we don't need to ignore the warning anymore: https://github.com/astropy/astropy/blob/9ec4ae327bc8ca20bebbdd3987cbc14995bf1ad6/pyproject.toml#L184

Reply:
@WilliamJamieson  LMK when the warning is removed, then I can review again!